src/showcase/showcase-changes.less

```diff
@@ -99,11 +98,11 @@
   }
 
   .y-axis {
-      .tick {
-          &.orient-right text {
-            .font(10px);
-          }
+    .tick {
+      &.orient-right text {
+        .font(10px);
```

Comment:
Why wouldn't it be? In any case I didn't change that, just updated the indentation.